typescript/appconfig-hosted-configuration-lambda-extension/README.md

```diff
@@ -0,0 +1,58 @@
+# AWS AppConfig Hosted Configuration with Lambda Extension
+<!--BEGIN STABILITY BANNER-->
+---
+
+![Stability: Stable](https://img.shields.io/badge/stability-Stable-success.svg?style=for-the-badge)
+
+> **This is a stable example. It should successfully build out of the box**
+>
+> This examples does is built on Construct Libraries marked "Stable" and does not have any infrastructure prerequisites to build.
+
+---
+<!--END STABILITY BANNER-->
+
+This an example of AppConfig fature toggle use case with a hosted configuration and a consuming Lambda using AppConfig Lambda extension integration.
```

Comment:
This would be a great place to include a brief description of why this pattern might be useful.
eg. "Deploying an AppConfig resource can allow for easier deployments to multiple environments from development to production and allow for easier changing of necessary configurations depending on environment ...."